<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="5" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-11T23:22:05 IST" name="Suite" finished-at="2025-03-11T23:22:21 IST" duration-ms="16522">
    <groups>
      <group name="smoke">
        <method signature="LoginTest.test1(java.lang.String,java.lang.String)[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" name="test1" class="NoMercy.Test.LoginTest"/>
      </group> <!-- smoke -->
      <group name="sanity">
        <method signature="LoginTest.test2()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" name="test2" class="NoMercy.Test.LoginTest"/>
      </group> <!-- sanity -->
    </groups>
    <test started-at="2025-03-11T23:22:05 IST" name="Test" finished-at="2025-03-11T23:22:21 IST" duration-ms="16522">
      <class name="NoMercy.Test.LoginTest">
        <test-method is-config="true" signature="launchHomePage()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:05 IST" name="launchHomePage" finished-at="2025-03-11T23:22:09 IST" duration-ms="3354" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchHomePage -->
        <test-method signature="test1(java.lang.String,java.lang.String)[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:09 IST" name="test1" data-provider="provideDataToTest" finished-at="2025-03-11T23:22:09 IST" duration-ms="443" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[anu@example.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Anu@1234]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method is-config="true" signature="terminateBrowser()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:09 IST" name="terminateBrowser" finished-at="2025-03-11T23:22:09 IST" duration-ms="152" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- terminateBrowser -->
        <test-method is-config="true" signature="launchHomePage()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:09 IST" name="launchHomePage" finished-at="2025-03-11T23:22:13 IST" duration-ms="4101" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchHomePage -->
        <test-method signature="test1(java.lang.String,java.lang.String)[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:13 IST" name="test1" data-provider="provideDataToTest" finished-at="2025-03-11T23:22:14 IST" duration-ms="464" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[anu@example.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Anu@1234]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method is-config="true" signature="terminateBrowser()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:14 IST" name="terminateBrowser" finished-at="2025-03-11T23:22:14 IST" duration-ms="134" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- terminateBrowser -->
        <test-method is-config="true" signature="launchHomePage()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:14 IST" name="launchHomePage" finished-at="2025-03-11T23:22:16 IST" duration-ms="2008" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchHomePage -->
        <test-method retried="true" signature="test2()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:16 IST" name="test2" finished-at="2025-03-11T23:22:17 IST" duration-ms="1316" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email o password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email o password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at NoMercy.Test.LoginTest.test2(LoginTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="terminateBrowser()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:17 IST" name="terminateBrowser" finished-at="2025-03-11T23:22:17 IST" duration-ms="169" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- terminateBrowser -->
        <test-method is-config="true" signature="launchHomePage()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:17 IST" name="launchHomePage" finished-at="2025-03-11T23:22:19 IST" duration-ms="1983" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchHomePage -->
        <test-method signature="test2()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:19 IST" name="test2" finished-at="2025-03-11T23:22:21 IST" duration-ms="1311" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email o password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email o password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at NoMercy.Test.LoginTest.test2(LoginTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:249)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="terminateBrowser()[pri:0, instance:NoMercy.Test.LoginTest@7f132176]" started-at="2025-03-11T23:22:21 IST" name="terminateBrowser" finished-at="2025-03-11T23:22:21 IST" duration-ms="149" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- terminateBrowser -->
      </class> <!-- NoMercy.Test.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
